packages/asc-assets/.svgo.yml

```diff
@@ -0,0 +1,3 @@
+plugins:
+  - removeViewBox: false
+  - removeDimensions: true
```

Comment:
There is a lot more optimisations we could potentially benefit from, like:
```yaml
- removeDoctype
- removeComments
- removeMetadata
- removeXMLNS
- removeEditorsNSData
- cleanupIDs
- removeRasterImages
- removeUselessDefs
- removeUnknownsAndDefaults
- removeUselessStrokeAndFill
- removeHiddenElems
- removeEmptyText
- removeEmptyAttrs
- removeEmptyContainers
- removeUnusedNS
- removeDesc
- prefixIds
- ...
```

Reply:
I didn't really dove into this in detail yet. I only added the optimisations I knew we would need for now. But would be great if we could use these! Do you have any experience with the optimisations you listed above?

Reply:
Most of them, yes.